spec/support/js_console.rb

```diff
@@ -0,0 +1,22 @@
+module JsConsole
+  def console_messages
+    page.driver.phantomjs_logger.rewind
+    page.driver.phantomjs_logger.read.split("\n")
+  end
+
+  def clear_console_messages
+    page.driver.phantomjs_logger.reopen if defined?(page)
```

Comment:
When is `page` not defined?

Shouldn't we rather limit the call to this method to the appropriate tests?
```ruby
config.before(:example, js: true) do
  ...
end
```